go/libraries/doltcore/sqle/sqlddl_test.go

```diff
@@ -83,11 +83,6 @@ func TestCreateTable(t *testing.T) {
 			query:       "create table dolt_table (id int primary key, age int)",
 			expectedErr: "Invalid table name",
 		},
-		{
-			name:        "Test bad table name begins with number",
```

Comment:
Rather than remove these tests, we should have tests confirming that these are valid names for tables.